components/zoho_crm/zoho_crm.app.mjs

```diff
@@ -112,6 +80,29 @@ export default {
         ...extraOpts,
       };
     },
+    _getLabelValueForModuleRecord(moduleType, record) {
+      const value = record.id;
+      let label;
+
+      if (moduleType === "Leads") {
+        label = record.Full_Name;
+      } else if (moduleType === "Accounts") {
+        label = record.Account_Name;
+      } else if (moduleType === "Contacts") {
+        label = record.Full_Name;
+      } else if (moduleType === "users?type=ActiveUsers") {
+        label = record.full_name;
+      } else if (moduleType === "Attachments") {
+        label = record.File_Name;
+      } else {
+        label = value;
+      }
```

Comment:
```suggestion
      const paramsObj = {
        Leads: "Full_Name",
        Accounts: "Account_Name",
        Contacts: "Full_Name",
        'users?type="ActiveUsers"': "full_name",
        Attachments: "File_Name",
      };
      const label = paramsObj[moduleType] ?? value;
```
Optional - another way to do it without repeating the assignments and comparisons.

Reply:
Thank you, this is MUCH better!